### ISSUE TITLE
Is it possible to exclude a block from linting?

### ISSUE THREAD

**Issue by craiga** [thibaudcolas/curlylint#36]
### Is your proposal related to a problem?

I have a block of code which is triggering #23. 

```
<span data-{{ triggers-issue}}>Hello, world!</span>
```

Before that issue is being fixed, I'd like to be able to just exclude that block from linting using something like a "noqa" comment.

### Describe the solution you’d like

I'm not sure what would work; perhaps something like the following?

```
{# curlylintignore #}
  <span data-{{ triggers-issue}}>Hello, world!</span>
{# endcurlylintignore #}
```

### Describe alternatives you’ve considered

I can ignore the entire file, but I'd like to be able to lint the rest of the file.

**Comment by tomasbackman** [comment]
Similar/same/very related with my new issue: #83

**Comment by dfrankow** [comment]
FYI my issue is intermingling HTML and Django template language, e.g.,

```
{% if something %}
<li>
{% endif %}

{# 100 lines of template #}

{% if something %}
</li>
{% endif %}
```

Yes, I could break this into two templates, but I'm just saying, this is the issue that comes up for me.

**Comment by PaulMarcantonio** [comment]
Some how when I corrected the <html lang> tag things started to flow...
<!DOCTYPE html>
<html lang="en-US">
perhaps this was the issue?  no sure but before that I had just <html lang="en">

I hope others find this helpful
